main.js

```diff
@@ -19,3 +20,15 @@ document.addEventListener('keydown', function (event) {
     $racecar.className = 'down';
   }
 });
+
+// users can start car
+var start = 0;
+function startCar(event) {
+  start += 5;
+  $racecar.style.left = start + 'px';
+  data.location.left = start;
+}
+
+document.addEventListener('keydown', function (event) {
+  setInterval(startCar, 16);
+});
```

Comment:
You shouldn't make a separate event handler for this, you should include it in your existing `keydown` event handler, and just check if the key that was pressed was a space bar